emulated_hue/api.py

```diff
@@ -319,17 +317,6 @@ async def async_group_action(self, request: web.Request, request_data: dict):
         # Create success responses for all received keys
         return send_success_response(request.path, request_data, username)
 
-    @routes.post("/api/{username}/groups")
-    @check_request()
-    async def async_create_group(self, request: web.Request, request_data: dict):
```

Comment:
The change of the groups logic is very different from the other component of this pr (allowing scene recalls for group 0). Can you create a separate pr for the other change instead of combining them into one like this?